chance.js

```diff
@@ -7356,27 +7378,27 @@ options,
         ],
         music_genres: {
             'general': [
-                'Rock', 
```

Comment:
Your commit has some unexpected changes to unrelated code.
Could you revert these unrelated changes?

Reply:
Reverted the unintentional changes!